### PR TITLE
Updating examples and instructions with new sync config format

### PR DESCRIPTION
**Pull request by hanssv**

### CODE REVIEW

File: docs/release-notes/RELEASE-NOTES-0.10.0.md [hunk 1]
```diff
@@ -2,8 +2,7 @@
 
 [This release][this-release] is focused on an encrypted Peer-to-Peer protocol and consistent serialization of chain objects.
 It:
-* Moves Peer-to-Peer communication from HTTP to encrypted channels (Noise is used to negoitiate cryptographic material).
-* Introduces a consistent and deterministic serialization using RLP, moving away from MessagePack and internal format.
```
Comment (gorillainduction) [inline docs/release-notes/RELEASE-NOTES-0.10.0.md]: Did you mean to remove the RLP line?

Comment (lucafavatella) [reply docs/release-notes/RELEASE-NOTES-0.10.0.md]: +1

File: docker/entrypoint.sh [hunk 2]
```diff
@@ -5,11 +5,11 @@ set -e
 EXTERNAL_PEER_ADDRESS="${EXTERNAL_PEER_ADDRESS:-http://$(curl -s https://api.ipify.org):3013/}"
 
 # Allow changing peers address[0], defaults to testnet node1
-PEERS_ADDRESS_0="${PEERS_ADDRESS_0:-http://31.13.249.1:3013/}"
+PEERS_ADDRESS_0="${PEERS_ADDRESS_0:-peer: {host: 31.13.249.1, port: 3015, pubkey: pp\$2eDAWTgveKp1C4dWhy9Hg59NCrg8TPUCKSXeEgvnPdro4ra177}"
 
 # Use TestNet example config as configuration template
 sed -e "s|peer_address: http://127.0.0.1:3013/|peer_address: ${EXTERNAL_PEER_ADDRESS}|g" \
-    -e "s|http://31.13.249.1:3013/|${PEERS_ADDRESS_0}|g" \
+    -e "s|peer: {host: 31.13.249.1, port: 3015, pubkey: pp\$2eDAWTgveKp1C4dWhy9Hg59NCrg8TPUCKSXeEgvnPdro4ra177}|${PEERS_ADDRESS_0}|g" \
```
Comment (lucafavatella) [inline docker/entrypoint.sh]: Does this mean that all running Docker containers will by default have the same peer identity?

File: docs/configuration.md [hunk 3]
```diff
@@ -36,15 +36,31 @@ You shall not share the private key (or the password) with anyone.
 
 The instructions below assume that:
 * The node is deployed in directory `/tmp/node`;
-* The initial network peer to join is "http://31.13.249.1:3013/".
+* The initial network peers to join are located at "31.13.249.1",
```
Comment (lucafavatella) [inline docs/configuration.md]: If so, can you please update list in RELEASE-NOTES-0.10.0.md ? cc @dincho

File: docker/entrypoint.sh (hunk 2 above)
Comment (lucafavatella) [reply docker/entrypoint.sh]: Ignore this please.

File: docs/release-notes/RELEASE-NOTES-0.10.0.md (hunk 1 above)
Comment (hanssv) [reply docs/release-notes/RELEASE-NOTES-0.10.0.md]: Yes, since you wrote the same thing a few lines further down...

**Comment by hanssv** [comment]
Will abandon this PR - all these changes with the new format will go in another PR...

**Comment by lucafavatella** [comment]
Superseded by https://github.com/aeternity/epoch/pull/916 (cc @hanssv )

**Pull request closed without merging**